info.go

```diff
@@ -59,12 +59,12 @@ type Info struct {
 }
 
 // Equivalent to HAProxy "show info" command.
-func (h *HAProxyClient) Info() (*Info, error) {
+func Info(h HAProxy) (*InfoResponse, error) {
```

Comment:
This is a breaking change to the existing API. It may be trivial but is worth noting.